datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -867,6 +867,16 @@ def check(self, instance):
         # type: (InstanceType) -> None
         raise NotImplementedError
 
+    def cancel(self):
+        # type: () -> None
+        """
+        This method is called when the check in unscheduled by the agent. This
+        is SIGNAL that the check is being unscheduled and can be called while
```

Comment:
All caps implies POSIX signals

```suggestion
        signals that the check is being unscheduled and can be called while
```